mk_mingw.mak

```diff
@@ -16,8 +16,8 @@ VPATH = . ./main ./parsers
 ctags.exe: OPT = -O4 -Os -fexpensive-optimizations
 ctags.exe: LDFLAGS = -s
 dctags.exe: OPT = -g
-dctags.exe: DEBUG = -DDEBUG
-dctags.exe: SOURCES += debug.c
+dctags.exe: CFLAGS += -DDEBUG
+# dctags.exe: SOURCES += $(MAIN_DIR)/debug.c
```

Comment:
Ah, missed that. Thanks for the clarification.

In that case there is no need to keep the line.